README.md

```diff
@@ -363,6 +363,86 @@ An optional unique identifier for the notification or an identifier for a batch
 </details>
 
 
+### Precompiled Letter
+
+#### Method
+
+The letter must contain:
+
+- A reference for the letter, used to identify the letter (this MUST NOT contain personal data)
+- The file content - a base64 encoded string 
+
+A pre-compiled letter can be sent in two ways: as a Java File object or a base64 string.
+If a File object is provided it is converted to a base64 string by the client.
+
+<details>
+<summary>
+Java File Object - Click here to expand for more information.
+</summary>
+
+```java
+File precompiledPDF = new File("<path to your file>");
+SendLetterResponse response = client.sendPrecompiledLetter("Your reference", precompiledPDF)
+
+```
+</details>
+
+<details>
+<summary>
+Base64 Ecoded String - Click here to expand for more information.
+</summary>
+
+```java
+String base64EncodedPDFFile = "<base64 encoded string of a file>";
+SendLetterResponse response = client.SendLetterResponse sendPrecompiledLetter("Your reference", base64EncodedPDFFile);
+```
+</details>
+
+#### Response
+
+If the request is successful, the SendLetterResponse is returned from the client. Attributes of the SendLetterResponse are listed below.
+
+<details>
+<summary>
+Click here to expand for more information.
+</summary>
+
+```java
+	UUID notificationId;
+	Optional<String> reference;
+	UUID templateId;
```

Comment:
Does this make sense if the templates for precompiled are `hidden`?

Reply:
Well its more one for the endpoint - but it was to keep it as similar to the a letter response as possible.